tests/layer_tests/tensorflow_tests/test_tf_Bucketize.py

```diff
@@ -4,90 +4,43 @@
 import numpy as np
 import pytest
 import tensorflow as tf
-from common.layer_test_class import check_ir_version
 from common.tf_layer_test_class import CommonTFLayerTest
 
-from unit_tests.utils.graph import build_graph
-
 
 class TestBucketize(CommonTFLayerTest):
-    def create_bucketize_net(self, input_shape, input_type, boundaries_size, ir_version,
-                             use_new_frontend):
-        """
-            Tensorflow net:                     IR net:
-                 Input            =>      Input        Boundaries
-                   |                           \       /
-               Bucketize                       Bucketize
-           {attrs: boundaries}
-        """
-
+    def _prepare_input(self, inputs_info):
+        assert 'input' in inputs_info, "Test error: inputs_info must contain `input`"
+        input_shape = inputs_info['input']
+        input_type = self.input_type
+        inputs_data = {}
+        input_data = np.random.randint(-20, 20, input_shape).astype(input_type)
```

Comment:
We need seriously think about this place. I strongly suggest to refactor original implementation.

It was an exceptional way to set an input in some unusual situations. Now I can see an overloading of _prepate_input in each PR. It looks like strange.

We may just introduce a dictionary to the parent class. Like inputs["name"] = [lval, rval, input_type] with default behavior in case a redefinition is absent for exact input.

Reply:
This is not a task of this activity. The most relevant thing is to have code coverage. For now, it does not create difficulties to create a custom function with data generation.